src/Money.ts

```diff
@@ -34,12 +40,12 @@ export default class Money {
   }
 
   /**
-   * Parse monetary string value
-   * @param string input
-   * @param string currency
+   * Create a Money object from a string representation of the amount of money
+   * @param mainAmount The quantity of money as a string in the main unit of the currency (e.g. '10.50' for £10.50)
+   * @param currency The alphabetic currency code as defined by ISO 4217 (e.g. GBP for Sterling)
    */
-  static fromString(input: string, currency: string) {
-    const amount = parseCurrencyString(input, currency)
+  static fromString(mainAmount: string, currency: string): Money {
```

Comment:
Can we rename mainAmount back to input here please.

Reply:
No problem can do, can I ask why `input` is preferred? (ee530e791ec2e14dddb3b4e5a4e4d41f51deb137)

Reply:
It is mostly because when we refer to the term `amount` it is a number, by using input we a creating a clear difference that it is a string representation input to a money object that needs parsing and converting to an amount.